_posts/multirepo_comments.adoc

```diff
@@ -0,0 +1,96 @@
+# The Dangers of Multirepo
+
+In this post I'm going to make the case for a somewhat unpopular opinion - using multiple git repos (AKA multirepo) is unnecessary and dangerous in many cases. A single git repo (AKA monorepo) is simpler and satisfies the requirements of many projects.
```

Comment:
I don't want to to mention where I saw these bad habits, don't want to make anyone look bad. Therefore, I rather keep it general.
This sentence is supposed to give the gist of the article so that readers can know right at the start if they want to read on or not. 
In light of these considerations, how do you think I should change the text?